MapboxSearch/gradle/versions.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     gradle_plugin = '7.3.1'
-    kotlin_version = '1.5.31'
-    kotlin_coroutines_version = '1.5.2'
+    kotlin_version = '1.9.21'
```

Comment:
It's better not to bump Kotlin version to latest because customers with older version of Kotlin and who can't bump it, won't be able to use the Search SDK.

Maps v11 is using `1.7.20`, it's better to use this version for search as well.